src/Illuminate/Foundation/Console/Kernel.php

```diff
@@ -294,6 +297,42 @@ protected function bootstrappers()
     }
 
     /**
+     * Set environment if 'env' option if provided.
+     *
+     * @return void
+     */
+    protected function getEnvironment()
+    {
+        foreach ($_SERVER['argv'] as $item) {
```

Comment:
This is a security issue. You need to first verify that we're running in the console.